tests/test.spec.js

```diff
@@ -0,0 +1,37 @@
+const { test, expect } = require('playwright/test')
+
+const login = 'standard_user';
+const password = 'secret_sauce';
+
+test.describe('Playwright Home Task 1', () => {
+
+  test.beforeEach(async ({ page }) => {
+
+    await page.goto('/');
+    await page.locator("#user-name").click();
+    await page.locator("#user-name").fill(login);
+    await page.locator("#password").click();
+    await page.locator("#password").fill(password);
+    await page.locator("#login-button").click();
+  });
+
+  test.afterEach(async ({ page }) => {
+    await page.close();
+  });
+
+  test('Test 1 - Perform Login:', async ({ page }) => {
+    await expect(page.locator(".title")).toHaveText('Products');
+    await expect(page.locator('#shopping_cart_container > a')).toBeVisible();
+    await expect(page.locator('#inventory_container > div > div:nth-child(2)')).toBeVisible();
```

Comment:
Do we have a better option to build this selector? :nth-child seems not a stable option.